### DISCUSSION TITLE
Connect to local websocket server in production?

### DISCUSSION THREAD

**Discussion by mahadiResonest** [tauri-apps/tauri#8992] [Q&A]
I have build a desktop app with Tauri-React. In production, this app will connect to a local websocket server (running on a embedded nuke). Both my tauri app and the nuke is in same router network. I cant connect my tauri app to that Nuke ws server in production?I havea proxyserver in the nuke as well, however i can connect with that via 
```json
 "allowlist": {
      "all": false,
      "app": {
        "all": true
      },
      "http": {
        "all": true,
        "request": true,
        "scope": ["http://192.168.11.17:3000/*", "http://192.168.11.17:8000/"]
      }
    },
```
this allowlist http config. I saw that it also bypass any http domain as well.  However, for ws, it doesnt work. Is there any way around?

**Comment by FabianLars** [comment]
You could try the websocket plugin: https://github.com/tauri-apps/plugins-workspace/tree/v1/plugins/websocket. This is basically the same as the http api but for websockets.

Alternatively you can play around with this setting https://tauri.app/v1/api/config/#securityconfig.dangeroususehttpscheme which should make it possible to use browser native websocket apis/libraries to connect with ws (without this setting only wss is allowed).
